templates/base/src/utils/Logger/logger.js

```diff
@@ -0,0 +1,65 @@
+
+import fetch from 'isomorphic-unfetch';
+
+/**
+ * USAGE: ->
+ * import Logger from '../utils/Logger';
+ *  call as :-
+ * Logger.error({ message: 'Here is the error' });
```

Comment:
Please apply this logger in places where we have console.log under template/base, please take latest

Reply:
done